tools/run-task

```diff
@@ -132,25 +113,22 @@ function listrify(steps, {concurrent = false, verbose = false} = {}) {
             if (Array.isArray(task)) {
                 return { title, task: () => listrify(task.map(_task => {
```

Comment:
`title` is the `listr` api, but it seemed weird to use that to describe a task in the task definition? 

`listrify` is really just the bridge between our tasks and what it expects. 

do you think it would be better to map to it more closely?